client/lib/domains/resolve-domain-status.js

```diff
@@ -131,11 +139,19 @@ export function resolveDomainStatus(
 					}
 				);
 				return {
-					statusText: translate( 'Verifying connection' ),
-					statusClass: 'status-verifying',
-					status: translate( 'Complete setup' ),
+					statusText: translate( 'Verifying' ),
+					statusClass: 'status-success',
+					status: translate( 'Verifying' ),
 					icon: 'verifying',
 					listStatusText: status,
+					noticeText: translate(
+						'It can take between a few minutes to 72 hours to verify the connection. You can continue to work on your site, but %(domainName)s won’t be reachable just yet.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'That's it! This will take between a few minutes to 72 hours. You can continue to work on your site, but %s won't be reachable just yet.' )` *ES Score: 6*